frontend/src/Student/components/PortfolioOtherMediaForm.js

```diff
@@ -0,0 +1,344 @@
+import React, { Component, Fragment } from "react";
```

Comment:
The notable thing about the new piece forms is that they disable changes on Year Level, Program, etc. if the portfolio already exists (after the first piece is added).